### PR TITLE
RPC based support for showing bgp routes and ip routes

### PR DESCRIPTION
**Pull request by mhampasagar**
RPC based support added for showing BGP routes and routes

As part of this added below SONiC YANG files, and the routes output is sent as a string
sonic-bgp-show.yang
sonic-ip-show.yang
Tested with 10K routes and time reduced more than half.

### CODE REVIEW

File: models/yang/sonic/sonic-bgp-show.yang [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            description
+                "Address family(IPv4/IPv6)";
+        }
+    }
+
+    output {
+        leaf status {
+            type string;
+            description
+                "Success / failure in BGP show execution request.";
```
Comment (venkatmahalingam) [inline models/yang/sonic/sonic-bgp-show.yang:48]: This should say...all BGP routes in JSON string format for success case.

File: models/yang/sonic/sonic-ip-show.yang [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                description
+                    "Clear BGP by prefix.";
+        }
+    }
+
+    output {
+        leaf status {
+            type string;
+            description
+                "Success / failure in ip route show execution request.";
```
Comment (venkatmahalingam) [inline models/yang/sonic/sonic-ip-show.yang:56]: all IP routes in JSON string format in case of success other failure string.

File: models/yang/sonic/sonic-ip-show.yang [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            }
+            default "IPv4";
+            description "Address family (IPv4/IPv6)";
+        }
+
+        leaf prefix {
+                type inet:ip-prefix;
+                description
+                    "Clear BGP by prefix.";
+        }
```
Comment (venkatmahalingam) [inline models/yang/sonic/sonic-ip-show.yang:49]: clear IP prefix or show ip prefix? not sure why this option needs to be handled in RPC

File: src/CLI/clitree/cli-xml/ipv4.xml [hunk 4]
```diff
@@ -64,7 +64,7 @@ limitations under the License.
 		            ptype="STRING">
                    <PARAM
 	                    help="Destination prefix"
-		                  name="A.B.C.D/M"
+		                  name="prefix"
```
Comment (venkatmahalingam) [inline src/CLI/clitree/cli-xml/ipv4.xml]: This is wrong, please revert this change and fix this using alternate method.

**Review by venkatmahalingam — APPROVED** [review]

File: models/yang/sonic/sonic-bgp-show.yang (hunk 1 above)
Comment (mhampasagar) [reply models/yang/sonic/sonic-bgp-show.yang:48]: Done
